provider/docker.go

```diff
@@ -256,6 +257,18 @@ func (provider *Docker) getIPAddress(container dockertypes.ContainerJSON) string
 		return "127.0.0.1"
 	}
 
+	if provider.UseBindPortIP {
+		port := provider.getPort(container)
+		for netport, portBindings := range container.NetworkSettings.Ports {
+			// FIXME(vdemeester) handle TCP/UDP
```

Comment:
Clearly this will need to be fixed prior to merge, but looks good other than that.